src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -1990,4 +1990,8 @@ Do you want to install the latest Dynamo update?</value>
 Next assemblies were loaded several times:
 </value>
   </data>
+  <data name="MessageFailedToAddFile" xml:space="preserve">
```

Comment:
Well, it seems in DynamoCoreWpf, most message box content message string has message- prefix.

E.g. MessageConfirmToInstallPackage -> "Are you sure you want to install {0} {1} ?"